src/modules/ui_handlers.js

```diff
@@ -0,0 +1,104 @@
+import { addListToLocalStorage } from './localStorage.js';
+
+const toDoLi = document.querySelector('.list');
+const parser = new DOMParser();
+
+const addToDo = (todo, todoList) => {
```

Comment:
I think your procedure in this function is great, but there are a lot of different processes in it. I think you could declare smaller functions and call them when required inside this function.